### ISSUE TITLE
Is LucyRTL8125Ethernet.kext broken with Opencore 0.62?

### ISSUE THREAD

**Issue by tristan-k** [SchmockLord/Hackintosh-Intel-i9-10900k-AsRock-Z490-Phantom-ITX-TB3#25]
It seems `LucyRTL8125Ethernet.kext` is broken with [OpenCore 0.62](https://www.hackintosh-forum.de/forum/thread/50619-opencore-0-62-und-lucyrtl8125ethernet-kext-hakt/). Can someone confirm or deny this?

There is a [workaround](https://www.reddit.com/r/hackintosh/comments/i3pega/z490_itx_guide/):

> Make sure you install the appropriate kext ( LucyRTL8125Ethernet.kext ) within your EFI folder and Ethernet should appear under System Preferences -> Network! Then you'll need to select Ethernet, click Advanced, click Hardware, and Select Configure : Manually with Speed : 1000baseT and wired internet should connect. With Configure : Automatically selected, as is default, wired internet will not connect. Verification can be made by viewing the Ethernet port on the back of the motherboard and inspecting the lights; no lights on means the port is not active.

![d48fcrnp71f51](https://user-images.githubusercontent.com/732025/98469502-5a852c00-21e0-11eb-8f34-24d6b353893e.png)

**Comment by lctncld** [comment]
I guess you have to use a different NIC for osx installation from recovery image? Since you can't configure these settings in the base installer.

**Comment by tristan-k** [comment]
> I guess you have to use a different NIC for osx installation from recovery image? Since you can't configure these settings in the base installer.

No, you can do the [changes ](https://www.insanelymac.com/forum/topic/343542-lucyrtl8125ethernetkext-for-realtek-rtl8125/?do=findComment&comment=2737983)in the terminal from within the recovery.

```
$ ifconfig -m 
$ ifconfig en0 media 100baseTX mediaopt half-duplex
```

**Comment by PFalko** [comment]
Is there any way to include this in the bootimage via opencore?
